webapp/components/display-logo.js

```diff
@@ -116,7 +116,7 @@ class DisplayLogo extends ProductInfo(PolymerElement) {
     if (!showSource || !product.labels) {
       return '';
     }
-    const sources = window.wpt.Sources;
+    const sources = Sources;
     return product.labels.find(s => sources.has(s));
```

Comment:
do you need the sources var (ie: can you just do Sources.has(s) below)?

Reply:
Nope; removed.